reference/api/bindings_api.md

```diff
@@ -9,7 +9,7 @@ Examples for bindings include ```Kafka```, ```Rabbit MQ```, ```Azure Event Hubs`
 
 - [Bindings Structure](#bindings-structure)
 - [Invoking Service Code Through Input Bindings](#invoking-service-code-through-input-bindings)
-- [Sending Messages to Output Bindings](#sending-messages-to-output-bindings)
+- [Sending Messages to Output Bindings](https://github.com/dapr/docs/blob/be9a77d4580ac925ed8461e3cc17173210c75c43/reference/api/bindings_api.md#invoking-output-bindings)
```

Comment:
Thanks for the quick response.. apologies, I wasn't very clear. the entire URL is not needed, you can just put the following: `#invoking-output-bindings`.

Reply:
Done - that's my fault for not putting two-and-two together